src/components/MainContent/parseUtils.tsx

```diff
@@ -23,18 +23,34 @@ export const NewLine: React.SFC = ({ children }) => (
 
 const replaceDashWithDot = (text: string) => text.replace('-   ', '• ');
 
+const removeBackSlash = (text: string) => text.replace(/(\\)([_<>])/g, '$2');
+
+const replacer = (match: string, code: string) => String.fromCharCode(Number(code));
+const replaceCharRef = (text: string) => text.replace(/&#(\d*)/g, replacer);
+
+const parsersInOrder = [replaceDashWithDot, removeBackSlash, replaceCharRef];
+const applyParsers = (parsers: ((text: string) => string)[], text: string) => {
+  let parsedText: string = text;
+  parsers.map((parser: (text: string) => string) => {
```

Comment:
`map` returns a new array when it completes. If you merely want to iterate over the array and execute a function on each item, use `foreach` instead.
```suggestion
  parsers.forEach((parser: (text: string) => string) => {
```

Reply:
I changed it from map to forEach. Thank you for the clear explanation.